fineract-provider/src/main/java/org/apache/fineract/portfolio/self/loanaccount/data/SelfLoansDataValidator.java

```diff
@@ -93,7 +93,7 @@ public void validateRetrieveTransaction(UriInfo uriInfo) {
 
         final String loanTypeParameterName = "loanType";
         final String loanTypeStr = this.fromApiJsonHelper.extractStringNamed(loanTypeParameterName, element);
-        baseDataValidator.reset().parameter(loanTypeParameterName).value(loanTypeStr).notNull().equals("individual");
```

Comment:
why remove this .equals("individual") validation? (Twice, both here, and below.)

Reply:
SpotBug Said: ```This method calls equals(Object) on two references of different class types and analysis suggests they will be to objects of different classes at runtime. Further, examination of the equals methods that would be invoked suggest that either this call will always return false, or else the equals method is not be symmetric (which is a property required by the contract for equals in class Object).```

My think was since the types are different..it always returns false...which maybe wrong..so i removed it.

I will add this back and suppress this for now..We probably need a better way to handle this in the near future.